### PR TITLE
feat(cov): add nyc instrument passthrough

### PR DESCRIPTION
**Pull request by yinseny**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

N/A

##### Description of change

add a --nyc instrument in cov.js, pass everything in it to nyc

### CODE REVIEW

File: lib/cmd/cov.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ class CovCommand extends Command {
         description: 'prerequire files for coverage instrument',
         type: 'boolean',
       },
+      nyc: {
+        description: 'nyc instruments passthrough',
+        type: 'string',
```
Comment (atian25) [inline lib/cmd/cov.js]: how about move `covArgs` to this default value ?

File: lib/cmd/cov.js [hunk 2]
```diff
@@ -116,6 +120,14 @@ class CovCommand extends Command {
       this.addExclude('**/*.d.ts');
     }
 
+    // nyc args passthrough
+    let nycArgs = context.argv.nyc;
+    delete context.argv.nyc;
```
Comment (atian25) [inline lib/cmd/cov.js]: prefer set to undefined

File: lib/cmd/cov.js (hunk 1 above)
Comment (yinseny) [reply lib/cmd/cov.js]: done

File: README.md [hunk 3]
```diff
@@ -170,6 +170,14 @@ You can pass any mocha argv.
 - `-x` add dir ignore coverage, support multiple argv
 - `--prerequire` prerequire files for coverage instrument, you can use this options if load files slowly when call `mm.app` or `mm.cluster`
 - `--typescript` / `--ts` enable typescript support, default to `false`, if true, will auto add `.ts` extension and ignore `typings` and `d.ts`.
+- `--nyc` nyc instruments passthrough. you can use this to overwrite egg-bin's default nyc instruments and add additional ones.
+  >if you want to add addtional nyc reporters, you need to use this rather than add `reporter` key in `.nycrc` because:
```
Comment (atian25) [inline README.md]: markdown style, if you want to use `ref`, should add space after `>`

or change to `  - if` for secondary list

**Comment by atian25** [comment]
thanks for your work, I'll handler the rest

**Comment by atian25** [comment]
4.8.0

**Pull request merged**